### DISCUSSION TITLE
Need help to Get OIDC access token on authorization code

### DISCUSSION THREAD

**Discussion by vijay-pandey11** [ory/.github#120] [Q&A]
Hi,

I have integrated Ory Kratos with our custom company-used OIDC provider (call it SAMAUTH). It works fine & I get auth token (access token returned to my react UI which integrates this SAMAUTH. 
ORY Kratos does not give me this SAMAUTH back when placed in my react UI..
For some business requirements I need to fetch the SAMAUTH issued authorization code (from React frontend), which is currently received by Kratos via the callback on the successful authorization. Is there any callback which I can hook and receive the authorization code?

Can you please advice how can I achieve the above?

Thanks in advance.

**Comment by vinckr** [comment]
Hello @vijay-pandey11 

It seems this is a duplicate of https://github.com/ory/kratos/discussions/4127

See my answer here: https://github.com/ory/kratos/discussions/4127#discussioncomment-12180557
